services/cartService.js

```diff
@@ -0,0 +1,32 @@
+const cartDao = require("../models/cartDao");
+
+const getCartList = async (userId) => {
+  const cartList = await cartDao.getCartList(userId);
+  return cartList;
```

Comment:
추가로 필요한 비즈니스 로직이 없기에 전반적으로 코드가 간단하고, 상우님도 깔끔하게 작성 잘해주셨네요!

다만 지금과 같이 작성해주신 코드 이외에 별다른 로직이 들어가지 않는다면 변수 선언과 비동기처리 없이 곧바로 return 해주시는것이 보다 조금 더 깔끔한 코드가 될 수 있습니다~!

나머지 모듈들도 일괄적으로 적용 및 반영 부탁드릴게요 👍 
```suggestion
  return cartDao.getCartList(userId);
```